README.md

```diff
@@ -23,8 +23,8 @@ define a common infrastructure for the bootloader, system flash layout on
 microcontroller systems, and to provide a secure bootloader that enables
 simple software upgrades.
 
-MCUboot is operating system and hardware independent and relies on
-hardware porting layers from the operating. Currently, mcuboot works
+MCUboot is hardware independent and relies on
```

Comment:
I'm not sure this is quite right, either, though. Although probably not commonly used, it is reasonable to build MCUboot use one operating system and the application with a different one.